imitation/util/rollout.py

```diff
@@ -84,11 +84,11 @@ def add_step(self, idx, step_dict: Dict[str, np.ndarray]):
     self.partial_trajectories[idx].append(step_dict)
 
 
-def generate(policy, env, *, n_timesteps=None, n_episodes=None
-             ) -> List[Dict[str, np.ndarray]]:
+def generate_trajectories(policy, env, *, n_timesteps=None, n_episodes=None
+                          ) -> List[Dict[str, np.ndarray]]:
   """
-  Generate old_obs-action-new_obs-reward tuples from a policy and an
-  environment.
+  Generate a list of old_obs-action-new_obs-reward trajectories from a policy
```

Comment:
Good catch on the incorrect docstring. The `old_obs-action-new_obs-reward` tuple part needs to be removed too, since now it's returning separate vectors of concatenated observations,  concatenated actions, and concatenated rewards.